backend/functions/src/utils/on-create-document.ts

```diff
@@ -0,0 +1,4 @@
+export function onCreateDocument(obj: object) {
```

Comment:
A function name should describe what the function does. `onCreateDocument` implies that this function runs when a document is created, which is true, but it does not tell what this function is doing.

Use a more appropriate name here.